tests/test_indieauth/test_relmeauth.py

```diff
@@ -15,4 +15,4 @@ def test_rel_me_auth_link_discovery(self, target, index):
 
         responses.add(responses.Response(method="GET", url=url, body=index))
 
-        assert target(url) == ["https://indieweb.social/@capjamesg"]
+        assert target(url) == ["https://sr.ht/~capjamesg/"]
```

Comment:
Why did this url change in the assert? As the body that we're returning for tests doesn't seem to have changed?